adam_core/observers/tests/test_state.py

```diff
@@ -37,191 +37,49 @@ def test_get_observer_state_OriginCodes():
         assert perturber_state == observer_state
 
 
-def test_get_observer_state_X05():
+expected_precision = {
+    "X05": {"r": 15, "v": 2},
+    "I41": {"r": 10, "v": 2},
+    "W84": {"r": 10, "v": 2},
+    "000": {"r": 10, "v": 2},
+    "500": {"r": 0.10, "v": 0.01},
+}
+
+
+@pytest.mark.parametrize("code", ["X05", "I41", "W84"])
+@pytest.mark.parametrize("origin", ["sun", "ssb"])
+def test_get_observer_state(code, origin):
```

Comment:
This file looks like it got a lot of changes, but really, this is just parametrization to reduce the repetition. It doesn't do anything novel.